lib/agent/index.js

```diff
@@ -453,6 +507,16 @@ class Agent {
     return attachment
   }
 
+  /**
+   * Format handler data by turning existing single supply into arrays if necessary
+   * @private
+   * @param   {Array<Command|Replacer|ReactCommand>|Command|Replacer|ReactCommand} data The handler data
+   * @returns {Array<Command|Replacer|ReactCommand>}                                    The formatted data
+   */
+  _formatHandlerData (data) {
+    if (data) return Array.isArray(data) ? data : [data]
```

Comment:
@alex-taxiera Would it be better for code clarity's sake to add `else return undefined`?